packages/diagrams/backend/sirius-components-diagrams/src/main/java/org/eclipse/sirius/components/diagrams/components/DiagramComponent.java

```diff
@@ -94,30 +94,31 @@ public Element render() {
                     return new Element(EdgeComponent.class, edgeComponentProps);
                 })
                 .toList();
-        // @formatter:on
 
         List<Element> children = new ArrayList<>();
         children.addAll(nodes);
         children.addAll(edges);
 
-        // @formatter:off
         Position position = optionalPreviousDiagram.map(Diagram::getPosition)
                 .orElse(Position.UNDEFINED);
 
 
         Size size = optionalPreviousDiagram.map(Diagram::getSize)
                 .orElse(Size.UNDEFINED);
 
-        DiagramElementProps diagramElementProps = DiagramElementProps.newDiagramElementProps(diagramId)
+        DiagramElementProps.Builder diagramElementPropsBuilder = DiagramElementProps.newDiagramElementProps(diagramId)
                 .targetObjectId(targetObjectId)
                 .descriptionId(diagramDescription.getId())
                 .label(label)
                 .position(position)
                 .size(size)
-                .children(children)
-                .build();
-        // @formatter:on
-        return new Element(DiagramElementProps.TYPE, diagramElementProps);
+                .children(children);
+
+        if (style != null) {
+            diagramElementPropsBuilder.style(style);
+        }
```

Comment:
No need to make it optional, most of our APIs are there to fail fast, we are asking for a style, we should receive one